### PR TITLE
[Fix] Fixed the modal screen fit issue w/signoff

### LINKED ISSUE #1803: [Catalog] Modals on Meshery Catalog are not fitting to the screen
Add or update the following [Meshery Catalog](https://meshery.io/catalog) item(s):

**Name:** Modal Screen Fit Issue  
**Type:** UI Bug  
**Compatibility:** All Browsers  
**Pattern ID:** N/A  
**image:** 
Screenshot
**Pattern Info:** Modals on the Meshery Catalog are not fitting to the screen. They are exceeding the screen boundaries, making the content not completely visible.  
**Pattern Caveats:** N/A  
**URL:** [Meshery Catalog](https://meshery.io/catalog)  
**Download link:** 

**Steps to Reproduce:**
1. Go to [Meshery Catalog](https://meshery.io/catalog).
2. Click on any item to open a modal.
3. Observe that the modal content exceeds the screen boundaries and is not fully visible.

**Expected Behavior:**
The modals should be fully visible and fit within the screen boundaries, ensuring that all content is accessible and readable.

**Environment:**
- Browser: Chrome
- Operating System: Windows

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](https://slack.meshery.io) for access.

### PR DESCRIPTION
**Pull request by kanishk6103**
**Description**

This PR fixes #1803

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -9,8 +9,10 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0)
     eventmachine (1.2.7)
-    ffi (1.16.3)
-    ffi (1.16.3-x64-mingw-ucrt)
+    ffi (1.17.0-arm64-darwin)
+    ffi (1.17.0-x64-mingw-ucrt)
+    ffi (1.17.0-x86_64-darwin)
+    ffi (1.17.0-x86_64-linux-gnu)
```
Comment (Ashparshp) [inline Gemfile.lock:15]: Reasons for updating the Gemfile.lock file?

Comment (kanishk6103) [reply Gemfile.lock:15]: I just did bundle install on my windows machine and it automatically installed the updated one.

Comment (kanishk6103) [reply Gemfile.lock:15]: do you want me to revert that? @Ashparshp

Comment (Ashparshp) [reply Gemfile.lock:15]: Yeah! please, if you can!

Comment (kanishk6103) [reply Gemfile.lock]: done! kindly check @Ashparshp

**Review by Ashparshp — CHANGES_REQUESTED** [review]
Test for mobile screens as I can see it's not working perfectly fine!

**Comment by kanishk6103** [comment]
> Test for mobile screens as I can see it's not working perfectly fine!

I checked for mobile screens, height wise they work fine but width wise they're cut-off, is that the issue you are talking about? @Ashparshp

**Comment by Ashparshp** [comment]
When you scroll it check for buttons they don't fit!

**Comment by kanishk6103** [comment]
> When you scroll it check for buttons they don't fit!

image

Does this look fine?

**Comment by vishalvivekm** [comment]
@kanishk6103 
Thank you for your contribution.
Let's discuss this on Websites' call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by kanishk6103** [comment]
> @kanishk6103 Thank you for your contribution. Let's discuss this on Websites' call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

sure

**Review by Ashparshp — COMMENTED** [review]
LGTM! But why are you creating so many commits? It seems like there are a lot of empty ones.

**Comment by kanishk6103** [comment]
> LGTM! But why are you creating so many commits? It seems like there are a lot of empty ones.

I am really sorry for that, when I commited the responsive behaviour of buttons I forgot to sign it, so I had to rebase which made a lot of commits. Other than that, the master branch was being updated so I was updating my branch which again created a bit more commits. I am sorry for that.

**Comment by kanishk6103** [comment]
> @kanishk6103, you've created 20 commits?

I did not mean to create them, as I said I forgot to sign, so I had to rebase and sign the previous ones, that created commits, rest are me updating my branch to the latest ones. Again, I am sorry for this, what should I do? @Ashparshp

**Comment by Ashparshp** [comment]
@kanishk6103, no worries at all! These things happen.... I was just reminding you in case you missed it. You can also ignore this for now... No sweat 😅. Keep up the good work!

**Comment by Ashparshp** [comment]
Also, If you want to clean up the commit history, you can squash your commits into one. This will help keep the history neat...

**Comment by leecalcote** [comment]
@kanishk6103 --> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

**Comment by sudhanshutech** [comment]
@kanishk6103 ?

**Comment by kanishk6103** [comment]
> @kanishk6103 ?

Hey! I think i should raise a new PR, this one is a bit messed up.

**Comment by kanishk6103** [comment]
I have created a new PR for this @Ashparshp

**Pull request closed without merging**